test/Microsoft.OpenApi.Trimming.Tests/Microsoft.OpenApi.Trimming.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is this hooked up to an action / CI step? That would be ideal so if new warnings were introduced, the PR validation fails.

Reply:
Just added it to the solution so the CI validates this project as well. 4cc030d1

Reply:
Just adding it to the solution isn't enough. It needs to be `dotnet publish`'d in order for the warnings to show.